README.md

```diff
@@ -84,6 +84,13 @@ The `config` file has system defaults which are used when building the image and
 
 - `UMBREL_BRANCH` - The git branch or commit SHA in the custom repo that should be checked out.
 
+**WIFI setup**
+[raspberrypi.org - setting up wireless networking](https://www.raspberrypi.org/documentation/configuration/wireless/headless.md)
+```
+cd /boot
+sudo nano wpa_supplicant.conf
+```
```

Comment:
This is hard to understand and just confusing, this guide is much better: https://umbrelinfo.github.io/troubleshooting.html#using-wifi-instead-of-ethernet

Reply:
> This is hard to understand and just confusing, this guide is much better: https://umbrelinfo.github.io/troubleshooting.html#using-wifi-instead-of-ethernet

That link is a 404 now; https://archive.ph/HHBS8

Reply:
> That link is a 404 now

Yes, I moved the page to https://info.umbrel.tech/troubleshooting.html#using-wifi-instead-of-ethernet